pkg/osquery/extension.go

```diff
@@ -788,6 +791,7 @@ func (e *Extension) writeResultsWithReenroll(ctx context.Context, results []dist
 }
 
 func getEnrollDetails(client Querier) (service.EnrollmentDetails, error) {
+	fmt.Println("seph 1")
```

Comment:
remove the `seph` `println`s in this file? I see 3 of them

Reply:
oops, I missed pushing up a commit. Yeah